casts/bars/src/constants.tsx

```diff
@@ -7,8 +7,9 @@ export const UIConstant = {
     elastiicWidthController: 600,
     titleMinimumFontScale: 0.7,
 
-    refreshControlHeight: 50,
-    refreshControlLoaderSize: 20,
+    refreshControlSize: 40,
```

Comment:
It was named `height` for a large title, to know how much of the height it takes :)